install_bootstrap

```diff
@@ -282,6 +282,9 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
         ############################
         # AWS questions            #
         ############################
+        aws_default_region='aws_default_region'
+        ask_question "What is your AWS Default Region?" "$AWS_DEFAULT_REGION" "AWS Default Region" $aws_default_region
```

Comment:
I don't know the answer to this, so I'm asking -- Is it really a default region, or is it **the** region? In other words does it prefer this region but fallback on another region if there is some issue with the default region.

This may be AWS terminology and may be their standard, but just looking how it's phrased, it seems slightly odd to me.

Also, can you update the README for this prompt.

Reply:
I just updated the readme, hopefully this makes it more clear